addons/web/static/src/legacy/js/views/res_partner_view_dialogs.js

```diff
@@ -0,0 +1,43 @@
+/** @odoo-module alias=web.ResPartnerFormViewDialog **/
+
+import { _t } from 'web.core';
+import Dialog from 'web.Dialog';
+import { FormViewDialog } from 'web.view_dialogs';
+
+
+const ResPartnerFormViewDialog = FormViewDialog.extend({
+
+    _setRemoveButtonOption(options, btnClasses) {
+        options.buttons.push({
+            text: options.removeButtonText || _t("Remove"),
```

Comment:
That's strange that "remove" leads to breaking the parent-id herarchy. I don't think this is clearer than an unlink. Maybe to check with LNA but that seems really strange.

Reply:
Before this, when one clicked on the 'Remove' button from the FormViewDialog it actually deleted the partner...
LNA wanted a way to remove it from the company without deleting it. Maybe we should choose another wording ?